statics/js/components/graph-layout.js

```diff
@@ -1503,15 +1503,12 @@ TopologyGraphLayout.prototype = {
   },
 
   nodeImg: function(d) {
-    if (nodeImgMap[d.metadata.Type]) {
-      return nodeImgMap[d.metadata.Type];
-    }
-
-    return intfImg;
+    return getNodeImage(d.metadata.Type);
```

Comment:
It is like this because it allows users to customize the WebUI with their own files thanks to the config file.